client/services/operation/operation.go

```diff
@@ -0,0 +1,53 @@
+package operation_service
```

Comment:
underscores in package names are not welcome :) there are also no reason in the _service suffix in my opinion. please refer to the golang package names convention for more details: https://go.dev/blog/package-names
you can name the package as `package operation` and then refer to it using the same expression if you want to make imported package name more specific:
```golang
        operation_service "github.com/lidofinance/dc4bc/client/services/operation"
````